### PR TITLE
{math}[system/system] MCR vR2021b.1

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/13a4f60a2a89fe0204162dbc9251c808 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/b8e8d97c485fc257b4ce87ce9dabf616 for a full test report.

**Comment by bedroge** [comment]
@robert-mijakovic looks like the source URL is incorrect?

File: easybuild/easyconfigs/m/MCR/MCR-R2021b.1.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+homepage = 'https://www.mathworks.com/products/compiler/mcr/'
+description = """The MATLAB Runtime is a standalone set of shared libraries
+ that enables the execution of compiled MATLAB applications
+ or components on computers that do not have MATLAB installed."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://ssd.mathworks.com/supportfiles/downloads/%%(version)s/Release/%s/deployment_files/'
+               'installer/complete/glnxa64/' % local_update]
+sources = ['MATLAB_Runtime_%(version)s_glnxa64.zip']
```
Comment (bedroge) [inline easybuild/easyconfigs/m/MCR/MCR-R2021b.1.eb:18]: I think this has to be:
```suggestion
sources = ['MATLAB_Runtime_%%(version)s_Update_%s_glnxa64.zip' % local_update]
```
?

File: easybuild/easyconfigs/m/MCR/MCR-R2021b.1.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+checksums = ['d7067b4f4977350395fd75de1aeefc9de39010fbdcf32d99a5f2e8955e74121e']
```
Comment (bedroge) [inline easybuild/easyconfigs/m/MCR/MCR-R2021b.1.eb:19]: This one also doesn't look correct, it's the same one as https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/m/MCR/MCR-R2021b.eb#L18.

**Review by bedroge — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/MCR/MCR-R2021b.1.eb (hunk 2 above)
Comment (robert-mijakovic) [reply easybuild/easyconfigs/m/MCR/MCR-R2021b.1.eb:19]: You are right, I messed it up as the URL was wrong.

**Comment by robert-mijakovic** [comment]
> @robert-mijakovic looks like the source URL is incorrect?

@bedroge Source URL is fixed.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @robert-mijakovic!

**Pull request merged**